ICE on log+base64encode code in an imported module

Take neofs-contracts and make a benign change:
```
--- a/common/storage.go
+++ b/common/storage.go
@@ -17,5 +17,6 @@ func GetList(ctx storage.Context, key interface{}) [][]byte {
 // SetSerialized serializes data and puts it into contract storage.
 func SetSerialized(ctx storage.Context, key interface{}, value interface{}) {
        data := std.Serialize(value)
+       runtime.Log("setserialized  " + std.Base64Encode(data))
        storage.Put(ctx, key, data)
 }
```
Now compile it and boom!
```
../neo-go/bin/neo-go contract compile -i alphabet -c alphabet/config.yml -m alphabet/config.json -o alphabet/alphabet_contract.nef
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xd82891]

goroutine 1 [running]:
github.com/nspcc-dev/neo-go/pkg/compiler.(*codegen).emitExplicitConvert(0xc0004deb40?, {0x1259158, 0x1942ec0}, {0x0, 0x0})
        github.com/nspcc-dev/neo-go/pkg/compiler/codegen.go:1467 +0x31
github.com/nspcc-dev/neo-go/pkg/compiler.(*codegen).Visit(0xc0004deb40, {0x12589d8?, 0xc0002b5300?})
        github.com/nspcc-dev/neo-go/pkg/compiler/codegen.go:989 +0x203d
go/ast.Walk({0x124fe20?, 0xc0004deb40?}, {0x12589d8?, 0xc0002b5300?})
        go/ast/walk.go:51 +0x62
go/ast.Walk({0x124fe20?, 0xc0004deb40?}, {0x1258b90?, 0xc000328cb0?})
        go/ast/walk.go:206 +0xa10
github.com/nspcc-dev/neo-go/pkg/compiler.(*codegen).Visit(0xc0004deb40, {0x1258988?, 0xc0000dd8c0?})
        github.com/nspcc-dev/neo-go/pkg/compiler/codegen.go:1219 +0x6dee
go/ast.Walk({0x124fe20?, 0xc0004deb40?}, {0x1258988?, 0xc0000dd8c0?})
        go/ast/walk.go:51 +0x62
github.com/nspcc-dev/neo-go/pkg/compiler.(*codegen).convertFuncDecl(0xc0004deb40, {0x1258c08, 0xc0000c2280}, 0xc0000dd8f0, 0xc000511d00?)
        github.com/nspcc-dev/neo-go/pkg/compiler/codegen.go:521 +0xa0b
github.com/nspcc-dev/neo-go/pkg/compiler.(*codegen).compile.func2(0xc0000c2280, 0xc000634e10)
        github.com/nspcc-dev/neo-go/pkg/compiler/codegen.go:2218 +0x3da
github.com/nspcc-dev/neo-go/pkg/compiler.(*codegen).ForEachFile.func1(0xc00051f2c0)
        github.com/nspcc-dev/neo-go/pkg/compiler/compiler.go:103 +0x82
github.com/nspcc-dev/neo-go/pkg/compiler.(*codegen).ForEachPackage(0xc0004deb40, 0xc000511e50)
        github.com/nspcc-dev/neo-go/pkg/compiler/compiler.go:94 +0xcf
github.com/nspcc-dev/neo-go/pkg/compiler.(*codegen).ForEachFile(0xe35580?, 0xc00047d3f8?)
        github.com/nspcc-dev/neo-go/pkg/compiler/compiler.go:100 +0x45
github.com/nspcc-dev/neo-go/pkg/compiler.(*codegen).compile(0xc0004deb40, 0xc0000dcea0, 0x1?)
        github.com/nspcc-dev/neo-go/pkg/compiler/codegen.go:2205 +0x458
github.com/nspcc-dev/neo-go/pkg/compiler.codeGen(0xc0000dcea0)
        github.com/nspcc-dev/neo-go/pkg/compiler/codegen.go:2259 +0x385
github.com/nspcc-dev/neo-go/pkg/compiler.CompileWithOptions({0x7fff7b22304d?, 0x3?}, {0x0?, 0x0?}, 0xc00044a460)
        github.com/nspcc-dev/neo-go/pkg/compiler/compiler.go:219 +0x65
github.com/nspcc-dev/neo-go/pkg/compiler.CompileAndSave({0x7fff7b22304d, 0x8}, 0xc00044a460)
        github.com/nspcc-dev/neo-go/pkg/compiler/compiler.go:235 +0x2de
github.com/nspcc-dev/neo-go/cli/smartcontract.contractCompile(0xc00038d4a0)
        github.com/nspcc-dev/neo-go/cli/smartcontract/smart_contract.go:436 +0x832
github.com/urfave/cli.HandleAction({0xe63da0?, 0x1120110?}, 0x7?)
        github.com/urfave/cli@v1.22.5/app.go:524 +0x50
github.com/urfave/cli.Command.Run({{0xfef043, 0x7}, {0x0, 0x0}, {0x0, 0x0, 0x0}, {0x10175b8, 0x27}, {0x103377f, ...}, ...}, ...)
        github.com/urfave/cli@v1.22.5/command.go:173 +0x65b
github.com/urfave/cli.(*App).RunAsSubcommand(0xc0003fca80, 0xc00038d1e0)
        github.com/urfave/cli@v1.22.5/app.go:405 +0x91b
github.com/urfave/cli.Command.startApp({{0xff054e, 0x8}, {0x0, 0x0}, {0x0, 0x0, 0x0}, {0x1018788, 0x28}, {0x0, ...}, ...}, ...)
        github.com/urfave/cli@v1.22.5/command.go:372 +0x6e7
github.com/urfave/cli.Command.Run({{0xff054e, 0x8}, {0x0, 0x0}, {0x0, 0x0, 0x0}, {0x1018788, 0x28}, {0x0, ...}, ...}, ...)
        github.com/urfave/cli@v1.22.5/command.go:102 +0x825
github.com/urfave/cli.(*App).Run(0xc0003fc8c0, {0xc00003e0b0, 0xb, 0xb})
        github.com/urfave/cli@v1.22.5/app.go:277 +0x8a7
main.main()
        ./main.go:12 +0x33
make: *** [Makefile:28: alphabet/alphabet_contract.nef] Ошибка 2
```
Looks like it's related to the fact that `common` is an imported package, because if you're to change `examples/runtime/runtime.go` to something like
```
@@ -42,7 +43,8 @@ func CheckWitness() bool {
 
 // Log logs the given message.
 func Log(message string) {
-       runtime.Log(message)
+       data := std.Serialize(message)
+       runtime.Log("prefix" + std.Base64Encode(data))
 }
 
 // Notify emits an event with the specified data.
```
It'll work fine.

--- a/pkg/compiler/codegen.go
+++ b/pkg/compiler/codegen.go
@@ -1118,7 +1118,12 @@ func (c *codegen) Visit(node ast.Node) ast.Visitor {
 			// Function was not found, thus it can only be an invocation of a func-typed variable or type conversion.
 			// We care only about string conversions because all others are effectively no-op in NeoVM.
 			// E.g. one cannot write `bool(int(a))`, only `int32(int(a))`.
-			if isString(c.typeOf(n.Fun)) {
+			var typ = c.typeOf(n.Fun)
+			if typ == nil {
+				c.prog.Err = fmt.Errorf("can't find function %q in package %s", name, c.currPkg.PkgPath)
+				return nil
+			}
+			if isString(typ) {
 				c.emitConvert(stackitem.ByteArrayT)
 			} else if isFunc {
 				c.emitLoadVar("", name)

--- a/pkg/compiler/codegen.go
+++ b/pkg/compiler/codegen.go
@@ -1049,6 +1049,10 @@ func (c *codegen) Visit(node ast.Node) ast.Visitor {
 				}
 			} else {
 				typ := c.typeOf(fun)
+				if typ == nil {
+					c.prog.Err = fmt.Errorf("can't find function %q in package %s", name, c.currPkg.PkgPath)
+					return nil
+				}
 				ast.Walk(c, n.Args[0])
 				c.emitExplicitConvert(c.typeOf(n.Args[0]), typ)
 				return nil

